src/Dfe.PlanTech.Application/Users/Helper/UserHelper.cs

```diff
@@ -41,18 +41,18 @@ public UserHelper(IHttpContextAccessor httpContextAccessor, IPlanTechDbContext d
             var reference = establishmentDto.Urn ?? establishmentDto.Ukprn;
 
             if (reference is null)
-                throw new InvalidOperationException("Establishment reference cannot be null");
+                return 1;
```

Comment:
what will returning 1 do here?

Reply:
Noted and will be fixed as part of Jim's work.